app/models/post.rb

```diff
@@ -11,6 +11,9 @@ def latest_comments
   end
 
   after_save :update_posts_counter
```

Comment:
- [OPTIONAL] It would be nice to change the logic in the `update_posts_counter` method because the total number of posts that I created in the `db/seeds.rb` file was 4 while the posts_counter returns 10. To get the accurate figure, I would advise you to change the logic. 😎 🌵 

#### Hint:

```ruby
def update_posts_counter
  author.update(posts_counter: author.posts.all.length)
end
```